### ISSUE TITLE
Consider minor changes to Light Spectrum dialog

### ISSUE THREAD

**Issue by terracoda** [phetsims/molecules-and-light#334]
Our consultants asked a couple of questions about the Light Spectrum dialog content in the final review sessions for MAL.
1. One consultant asked, "Is this Alt Text?" when reading through the "Show Light Spectrum" dialog. She was wondering if there was a diagram that was being described. The presence of the diagram is not obviously apparent in the non-visual experience. I'd like to consider a label change to the dialog's button:
    - Show Light Spectrum
    - Light Spectrum Diagram
    - Light Spectrum Details

2. One consultant really liked the description of the “tiny range” rainbow for visible light, but wondered if we should describe the colors of a typical ROYGBIV rainbow.
    - Shown as a rainbow of colors (dark red to red to orange to yellow to green to blue to violet).
    - Shown as a rainbow of colors (red to orange to yellow to green to blue to indigo to violet).

**Comment by terracoda** [comment]
@emily-phet, @arouinfar, @BLFiedler, and @wandita, we will discuss these options for minor changes to the Light Spectrum diagram at tomorrow's design meeting.

**Comment by arouinfar** [comment]
@terracoda I like "Light Spectrum Diagram" for the button label.

For the rainbow, do we need to name all of the colors, or could we say something like:
* Shown as a rainbow of colors (red to violet).

 I am not sure that we need to describe the exact order of the colors in a rainbow to meet the goals of the diagram. By naming the ending colors, the user can still make the connection that red light has lower energy/frequency and longer wavelength while violet has a higher energy/frequency and a shorter wavelength.

**Comment by jessegreenberg** [comment]
We discussed on 5/5/20 - 

One drawback to changing the visual label is that we may want translators to re-translate the string. But we are adding other strings to this version so we may want to notify translators that there are new strings to translate anyway.

**Comment by terracoda** [comment]
@arouinfar asked:

> For the rainbow, do we need to name all of the colors?

I'm not a 100% sure, but it like good for BVI learners to describe the range and spell out the details that are visually available. So this might be best:

- Shown as a rainbow starting with red and ending in violet (red, yellow, green, blue, indigo, violet).

I'll see how this fits in with the rest of the description.

**Comment by arouinfar** [comment]
> Shown as a rainbow starting with red and ending with violet (red, yellow, green, blue, indigo, violet).

I think this is a good compromise. If users want to listen to the entire spectral order they can, but the most important information is fore-fronted.

**Comment by terracoda** [comment]
@arouinfar, I noticed in the design doc, the original description also included "from low to high energy":
- Shown **from low energy to high energy** as a rainbow of colors (dark red to red to orange to yellow to green to blue to violet).

I think @BLFiedler and I decided to remove that part as it is already stated at the top that the entire list is "lowest to highest energy."

I agree, I think the latest revision below and in https://github.com/phetsims/molecules-and-light/issues/334#issuecomment-624303676 is a good compromise:

@jessegreenberg, assigning to you to makes changes:
1. Change label:
    - Light Spectrum Diagram
2. Update the visible light description: 
    - Shown as a rainbow starting with red and ending with violet (red, yellow, green, blue, indigo, violet).
